### PR TITLE
feat: added example with kafka ingester

### PR DESCRIPTION
**Pull request by david-vavra**
Built on top of https://github.com/seznam/slo-exporter/pull/48

### CODE REVIEW

File: docs/modules/kafka_ingester.md [hunk 1]
```diff
@@ -41,6 +54,6 @@ For every received message from Kafka:
 ```
 
 Strictly speaking, none of the keys is mandatory, however please note that:
-- Event with Quantity=0 is basically noop for Producer module. To give an example, prometheusExporter does not increment any SLO metric for such events.
-- Event with empty Metadata does not allow much logic in sloEventProducer module. Also you need to take into account that all generated SLO events will have empty event_key.
-- In case you want allow ingesting events without SLO classification, you need to make sure that all events are classified within rest of the SLO exporter pipeline.
+- Event with explicitly set Quantity=0 is basically noop for Producer module. To give an example, prometheusExporter does not increment any SLO metric for such events.
```
Comment (lksv) [inline docs/modules/kafka_ingester.md:57]:
```suggestion
- Event with explicitly set `quantity`=0 is basically noop for Producer module. To give an example, prometheusExporter does not increment any SLO metric for such events.
```

File: examples/kafka/docker-compose.yaml [hunk 2]
```diff
@@ -0,0 +1,96 @@
+version: '3'
```
Comment (lksv) [inline examples/kafka/docker-compose.yaml:1]: Suggest to keep kafka setup as simple as possible - e.g. 1 zookeeper. e.g. https://github.com/conduktor/kafka-stack-docker-compose/blob/master/zk-single-kafka-single.yml

File: docs/modules/kafka_ingester.md [hunk 3]
```diff
@@ -16,21 +16,34 @@ logKafkaEvents: false
...
 groupId: ""
+# commitInterval indicates the interval at which offsets are committed to the broker.
+# If 0 (default), commits will be handled synchronously.
+commitInterval: <duration> # e.g. 0, 5s, 10m
+# retentionTime optionally sets the length of time the consumer group will be saved by the broker.
+# Default: 24h
+retentionTime: <duration>
+# lastOffsetAsDefault determines from whence the consumer group should begin consuming when it finds a partition without a committed offset.
+# If true, partition will be read from its LastOffset, otherwise from the FirstOffset.
+lastOffsetAsDefault: false
```
Comment (lksv) [inline docs/modules/kafka_ingester.md:30]: Suggest `kafkaStartOffset` with possible values `lastOffset` and `FirstOffset`. This naming will be also straightforward for kafka "experts".

File: pkg/kafka_ingester/kafka_ingester.go [hunk 4]
```diff
@@ -172,10 +208,53 @@ func (k *KafkaIngester) Run() {
...
 
+func getSchemaVersionFromHeaders(headers []kafka.Header) (string, bool) {
+	var (
+		found     bool
+		schemaVer string
+	)
+	for _, header := range headers {
+		if header.Key == schemaVersionMessageHeader {
+			schemaVer = string(header.Value)
+			found = true
```
Comment (lksv) [inline pkg/kafka_ingester/kafka_ingester.go:219]: `return schemaVer, true`?

File: pkg/kafka_ingester/kafka_ingester.go [hunk 5]
```diff
@@ -172,10 +208,53 @@ func (k *KafkaIngester) Run() {
...
+			break
+		}
+	}
+	return schemaVer, found
```
Comment (lksv) [inline pkg/kafka_ingester/kafka_ingester.go:223]: `return nil, false`?

File: pkg/kafka_ingester/kafka_ingester.go [hunk 6]
```diff
@@ -172,10 +208,53 @@ func (k *KafkaIngester) Run() {
...
+		quantity = 1
+	} else {
+		quantity = *ingestedEvent.Quantity
+	}
+	outputEvent := event.Raw{
+		Metadata:          ingestedEvent.Metadata,
+		Quantity:          quantity,
+		SloClassification: ingestedEvent.SloClassification,
+	}
+	return &outputEvent, err
```
Comment (lksv) [inline pkg/kafka_ingester/kafka_ingester.go:257]:
```suggestion
	return &outputEvent, nil
```

File: pkg/kafka_ingester/kafka_ingester.go (hunk 4 above)
Comment (david-vavra) [reply pkg/kafka_ingester/kafka_ingester.go:223]: Done as 89c41f8 within the primary PR https://github.com/seznam/slo-exporter/pull/48/files

File: pkg/kafka_ingester/kafka_ingester.go (hunk 5 above)
Comment (david-vavra) [reply pkg/kafka_ingester/kafka_ingester.go:227]: https://github.com/seznam/slo-exporter/pull/49/files#r599946338

File: pkg/kafka_ingester/kafka_ingester.go (hunk 6 above)
Comment (david-vavra) [reply pkg/kafka_ingester/kafka_ingester.go:261]: https://github.com/seznam/slo-exporter/pull/49/files#r599946338

File: examples/kafka/docker-compose.yaml (hunk 2 above)
Comment (david-vavra) [reply examples/kafka/docker-compose.yaml:1]: I wanted to example scenario to be as close to the real word setup as possible.

I prefer to keep it as it is now.

File: docs/modules/kafka_ingester.md (hunk 3 above)
Comment (david-vavra) [reply docs/modules/kafka_ingester.md]: https://github.com/seznam/slo-exporter/pull/48#discussion_r599936537

File: docs/modules/kafka_ingester.md (hunk 1 above)
Comment (david-vavra) [reply docs/modules/kafka_ingester.md]: Fixed, thanks.

**Review by FUSAKLA — APPROVED** [review]
:+1:

**Pull request merged**